src/bun.js/bindings/Path.cpp

```diff
@@ -184,27 +185,30 @@ static JSC::JSObject* createPath(JSGlobalObject* globalThis, bool isWindows)
             Path_functionToNamespacedPath, ImplementationVisibility::Public),
         0);
 
-    if (isWindows) {
-        path->putDirect(vm, clientData->builtinNames().sepPublicName(),
-            JSC::jsString(vm, WTF::String("\\"_s)), 0);
-        path->putDirect(vm, clientData->builtinNames().delimiterPublicName(),
-            JSC::jsString(vm, WTF::String(";"_s)), 0);
-    } else {
-        path->putDirect(vm, clientData->builtinNames().sepPublicName(),
-            JSC::jsString(vm, WTF::String("/"_s)), 0);
-        path->putDirect(vm, clientData->builtinNames().delimiterPublicName(),
-            JSC::jsString(vm, WTF::String(":"_s)), 0);
-    }
-
     return path;
```

Comment:
Can we use the added JSC_zig function helpers here to cleanup the creation of the zig to c++ methods?

Reply:
i inlined these constant strings right into the js code